makefile

```diff
@@ -31,6 +31,7 @@ CFLAGS = -O2
 #CC = gcc -O4 -Wall -pedantic -fno-strict-aliasing
 #CC = gcc -fprofile-arcs -ftest-coverage # then gcov f1.c; cat f1.c.gcov
 CC = gcc -g -Wall -pedantic 
+HOSTCC = gcc -g -Wall -pedantic
 
```

Comment:
I might suggest:
    `HOSTCC = gcc -g -Wall -pedantic`
    `CC = $(HOSTCC)     # change this is cross-compiling.`

Reply:
done!